Dockerfile.windows

```diff
@@ -119,9 +130,14 @@ RUN go build -o bin/google-cloud-ops-agent.exe ./cmd/ops_agent_windows; `
 # Packaging
 ###############################################################################
 
+FROM base as packaging
+
 WORKDIR /work
 
-RUN go get github.com/google/googet/v2/goopack;
+COPY . /work
+
+COPY --from=fluentbit /work/out/bin /work/out/bin
```

Comment:
[Optional] We should be able to copy the whole `/work/out` from the `fluentbit` container, right? For symmetry, if nothing else?..